R/Learner.R

```diff
@@ -572,6 +572,11 @@ Learner = R6Class("Learner",
         fallback_id = mlr_reflections$learner_fallback[[self$task_type]]
         if (!is.null(fallback_id)) {
           self$fallback = lrn(fallback_id, predict_type = self$predict_type)
+
+          if (self$predict_type == "quantiles") {
```

Comment:
Uh, is this really required here in the Learner class?

Reply:
Moved to `set_fallback()` function